### PR TITLE
Open-source section

### PR DESCRIPTION
**Pull request by RalucaRbc**

### CODE REVIEW

File: src/_component/config/reusable.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 %rounder-border-list { 
   border: 2px solid #707070;
-  width: 80%;
+  width: 85%;
```
Comment (M00N89) [inline src/_component/config/reusable.scss]: very well ! Raluca

**Review by M00N89 — COMMENTED** [review]
keep on !

**Review by MariaTeo — COMMENTED** [review]
Great work

**Review by andreimaff — APPROVED** [review]
nice job!

**Review by blowymind99 — APPROVED** [review]
Great work!

**Pull request merged**